gems/quilt_rails/README.md

```diff
@@ -326,67 +326,42 @@ function App() {
 export default App;
 ```
 
-##### Example: sending headers from Rails controller
+##### Example: Sending headers and custom data from Rails controller
```

Comment:
Should these maybe be two different sections?

Reply:
Yah, they should. Good catch

Reply:
Tried separating these. Lots of redundancy so I just kept them together.